src/rdkafka_admin.c

```diff
@@ -7952,11 +7972,13 @@ rd_kafka_ConsumerGroupDescription_new(
  *         Use rd_kafka_ConsumerGroupDescription_destroy() to free when done.
  */
 static rd_kafka_ConsumerGroupDescription_t *
-rd_kafka_ConsumerGroupDescription_new_error(const char *group_id,
-                                            rd_kafka_error_t *error) {
+rd_kafka_ConsumerGroupDescription_new_error(
+    const char *group_id,
+    rd_kafka_error_t *error,
+    rd_kafka_consumer_group_type_t type) {
```

Comment:
Let's remove this parameter, see :
```suggestion
    ) {
```